control/xferfcn.py

```diff
@@ -69,7 +69,14 @@
 
 
 # Define module default parameter values
-_xferfcn_defaults = {}
+_xferfcn_defaults = {
+    'xferfcn.display_format': 'poly',
+    'xferfcn.floating_point_format': '.4g'
+}
+
+def _float2str(value):
```

Comment:
I would suggest against doing this. It's anti-pattern for Python and also there is a simpler way to grab the default at the code level and reuse. 

```python
>>> _num_format = config.defaults.get('xferfcn.floating_point_format', ':.4g')
>>> print(f"Bla {0.232654684684654654:{_num_format}}")
Bla 0.232655
```

So you don't need to inject f-string machinery to the string representation.

Reply:
@henklaak If you can fix up this final issue, I think this PR is ready to post.